src/notifications/generate.ts

```diff
@@ -183,7 +183,7 @@ export const generateNotificationMap: Record<
       .avatarSource(ctx.source)
       .avatarUser(ctx.user)
       .referencePostModeration(ctx.post)
-      .targetSourceModeration(ctx.source),
+      .targetSourceModeration(),
```

Comment:
I think we can just inline this. We have no other place that uses this `targetSourceModeration` which we can also completely remove now.

```suggestion
      .targetUrl(`${process.env.COMMENTS_PREFIX}/squads/moderate`),
```

Reply:
We do actually. We also use it here:

```
  source_post_rejected: (builder, ctx: NotificationPostModerationContext) =>
    builder
      .icon(NotificationIcon.Bell)
      .referencePostModeration(ctx.post)
      .targetSourceModeration(ctx.source)
      .description(ctx.post.moderatorMessage ?? '')
      .avatarSource(ctx.source),

Reply:
Oh, nice catch. Ignore me with this one then.